src/lib/github/issue-presenter.ts

```diff
@@ -32,7 +32,7 @@ const formattedIssueComment = (data: RepoIssue[]): Issue[] => {
     return arrComments.map( (obj: IssueWithComments) => {
           return ({
             title: obj.title,
-            body: obj.comments.join(" -[] ")
+            body: obj.comments.map( (comment: Comment) => `-[]${comment}`).join(" ")
```

Comment:
You are missing end of line character. `\n`
So it should be `-[ ]${comment}\n`

Could you also add a space btn the brackets. Such that it is `[ ]`